lib/debug/thread_client.rb

```diff
@@ -394,7 +394,7 @@ def show_object_info expr
       end
     end
 
-    def add_breakpoint args
+    def initialize_breakpoint args
```

Comment:
I was confused by this name and it doesn't actually "add" a breakpoint to anything. so I think it's good to rename it.

Reply:
`make`?

Reply:
👍 updated